### DISCUSSION TITLE
How to make ColorPicker tab accessible when it nest inside of CommandBar

### DISCUSSION THREAD

**Discussion by yutingx** [microsoft/fluentui#27044] [Q&A] [answered]
I'm trying to make a toolbar that are using commandBar, at one of the item I made a `CommandBarButton` that can open a `ColorPicker` once clicked. Mouse access is fine, but keyboard can't smoothly access the slider of ColorPicker. I can use -> arrow key to navigate to next slider, but have to move the cursor all the way to the right for it. Tab will move focus back to button, which will close the colorPicker.

Any way to workaround this?
![image](https://user-images.githubusercontent.com/23161024/222605338-d23bf6dc-e4f2-4476-a494-ef3557375647.png)

**Comment by ValentinaKozlova** [comment, marked as answer]
Hello @yutingx, you might want to try a new V9 ColorPicker. It should be accessible:
https://react.fluentui.dev/?path=/docs/components-colorpicker--docs
